test/features/sats/sats_period_mat_provided_range.feature

```diff
@@ -179,4 +179,25 @@ Feature: [SAT-PM-P] Satellites Loaded using Period Materialization for provided
       | md5('1002') | md5('1995-08-07\|\|1002\|\|BEAH')    | Beah          | 1995-08-07   | 2019-05-05     | 2019-05-05 | *      |
       | md5('1003') | md5('1990-02-03\|\|1003\|\|CHRIS')   | Chris         | 1990-02-03   | 2019-05-05     | 2019-05-05 | *      |
       | md5('1004') | md5('1992-01-30\|\|1004\|\|DAVID')   | David         | 1992-01-30   | 2019-05-05     | 2019-05-05 | *      |
-      | md5('1010') | md5('1991-03-25\|\|1010\|\|JENNY')   | Jenny         | 1991-03-25   | 2019-05-05     | 2019-05-05 | *      |
\ No newline at end of file
+      | md5('1010') | md5('1991-03-25\|\|1010\|\|JENNY')   | Jenny         | 1991-03-25   | 2019-05-05     | 2019-05-05 | *      |
+
+  @fixture.satellite
+  Scenario: [SQLS-SAT-PM-P-05] Incremental load of a satellite with multiple timestamps in the same day loads all records, datetimes with 6 decimal places
```

Comment:
We do not need platform specific (`SQLS-`) prefixes anymore

Reply:
Corrected.